data/tests/fwupdtool.sh

```diff
@@ -45,6 +45,11 @@ echo "Showing hwids"
 run hwids
 rc=$?; if [ $rc != 0 ]; then error $rc; fi
 
+# ---
+echo "Showing security"
+run security
```

Comment:
In practice I don't think this is going to change _much_ because we don't run/support HSI in containers/VMs.  How about also adding some code to allow HSI to run when `CI` is set in env variables?

Reply:
Actually how about `#ifdef SUPPORTED_BUILD` instead?

Reply:
It's really to get the security mechanisms tested as part of coverage; the result itself isn't terribly interesting, but the fact we could return *something* really helps test coverage.

Reply:
Yeah; but with either of my suggestions it will launch more code in CI too.

Reply:
I looked at this; we do run the fwupd**tool** security checks -- we don't run the fwupd**mgr** ones tho as we check for VM/container in `fu-dbus-daemon.c` rather than lower down.

Reply:
Ah good. Thanks!